src/neuroglancer/datasource/precomputed/frontend.ts

```diff
@@ -112,6 +121,44 @@ export class MultiscaleVolumeChunkSource implements GenericMultiscaleVolumeChunk
   }
 }
 
+
+export class SkeletonSource extends BaseSkeletonSource {
+  get skeletonVertexCoordinatesInVoxels() {
+    return false;
```

Comment:
Add a "not implemented" message?

Reply:
if (source.skeletonVertexCoordinatesInVoxels) {
      mat4.scale(objectToDataMatrix, objectToDataMatrix, this.voxelSizeObject.size);
}
This is where this is used